### PR TITLE
 feat:[CAR-3720] Added NumberField

### PR DESCRIPTION
**Pull request by tokarevfedor1991**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -167,6 +167,7 @@
     "react-docgen-typescript-loader": "3.7.2",
     "react-dom": "17.0.2",
     "react-element-to-jsx-string": "14.3.4",
+    "react-number-format": "^4.9.4",
```
Comment (timurhamitov) [inline package.json:170]: @tokarevfedor1991 Can we make the package to be the exact version? Without the "^" symbol.

File: packages/ui/src/number-field/NumberField.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import { UseState } from '@superdispatch/ui-docs';
+import { NumberField } from '..';
+
+export default { title: 'Inputs/NumberField', component: NumberField };
+
+export const basic = () => (
+  <UseState initialState={false}>
+    {(value, setValue) => (
+      <NumberField
+        value={value}
```
Comment (timurhamitov) [inline packages/ui/src/number-field/NumberField.stories.tsx:10]: @tokarevfedor1991 Is it ok to use false as a default value for NumberField?

File: package.json (hunk 1 above)
Comment (tokarevfedor1991) [reply package.json:170]: ok)

File: packages/ui/src/number-field/NumberField.stories.tsx (hunk 2 above)
Comment (tokarevfedor1991) [reply packages/ui/src/number-field/NumberField.stories.tsx:10]: we can set false, but there will be nothing, not even 0

Comment (timurhamitov) [reply packages/ui/src/number-field/NumberField.stories.tsx:10]: I'm thinking about situations when we will refer to these stories to copy/paste the code. And to make it clearer for future users, we can set the default value that we are using for this field in Carrier TMS.

**Review by timurhamitov — APPROVED** [review]

File: packages/ui/src/number-field/NumberField.stories.tsx (hunk 2 above)
Comment (tokarevfedor1991) [reply packages/ui/src/number-field/NumberField.stories.tsx:10]: I will set '0' as default

Comment (timurhamitov) [reply packages/ui/src/number-field/NumberField.stories.tsx:10]: Can you check what is used in Carrier TMS? But I think it's not critical=)

**Pull request merged**